docs/schema/aws.md

```diff
@@ -2158,3 +2174,80 @@ Representation of an AWS [API Gateway Resource](https://docs.aws.amazon.com/apig
         ```
         (APIGatewayRestAPI)-[RESOURCE]->(APIGatewayResource)
         ```
+
+## AutoScalingGroup
+
+Representation of an AWS [Auto Scaling Group Resource](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html).
```

Comment:
Thanks for including this. Can you also add a link to the AWS API doc?

Reply:
Done